src/asf_tools/hydrosar/flood_map.py

```diff
@@ -111,19 +116,24 @@ def logstat(data: np.ndarray, func: Callable = np.nanstd) -> Union[np.ndarray, f
     return np.exp(st)
 
 
-def estimate_flood_depth(label, hand, flood_labels, estimator='iterative', water_level_sigma=3.,
-                         iterative_bounds=(0, 15), minimization_metric: str = 'fmi'):
+def estimate_flood_depth(label: int, hand: np.ndarray, flood_labels: np.ndarray, estimator: str = 'iterative',
+                         water_level_sigma: float = 3., iterative_bounds: Tuple[int, int] = (0, 15),
```

Comment:
(same question here: Should we use 15 or 6 for the time being - I won't leave comments on every occurrence of this)

Reply:
(same as :point_up: )